src/main/resources/create.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE customers (
+  id SERIAL NOT NULL, 
+  name varchar(255) NOT NULL, 
+  surname varchar(255) NOT NULL, 
+  age SMALLINT NOT NULL, 
+  PRIMARY KEY (id)
```

Comment:
Why you use that way? Not
id SERIAL PRIMARY KEY

Reply:
Yes you're right, actually there was constraint statement and I removed this word